src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -226,12 +229,25 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
         artifact_prefix=reproducers_dir,
         extra_env=options.extra_env)
 
-    if (not environment.get_value('USE_MINIJAIL') and
-        fuzz_result.return_code == constants.LIBFUZZER_ERROR_EXITCODE):
+    project_qualified_fuzzer_name = (
+        data_types.fuzz_target_project_qualified_name(
+            utils.current_project(), os.path.basename(target_path)))
+    dict_error_match = DICT_PARSING_FAILED_REGEX.match(fuzz_result.output)
```

Comment:
will this always be the very first thing in the output? i.e. if this occurs in the second line we won't catch it.

Reply:
Good catch, changed to re.search.